it/lessons/basics/basics.md

```diff
@@ -219,13 +219,13 @@ iex> 2 === 2.0
 false
 ```
 
-Una funzionalità importante di Elixir è che due valori di qualsiasi tipo possono essere confrontati, questo è particolarmente utile per l'ordinamento. Non abbiamo bisogno di memorizzare la sequenza di ordinamento, ma è importante esserne al corrente:
+Una funzionalità importante di Elixir è che due valori di qualsiasi tipo possono essere confrontati, questo è particolarmente utile per l'ordinamento. Non abbiamo bisogno di memorizzare l'ordine di precedenza, ma è importante esserne al corrente:
```

Comment:
I would probably expand the sentence "Non abbiamo bisogno di memorizzare l'ordine di precedenza, ma è importante esserne al corrente:" to explain it's talking about comparisons.In other words, I'd be more explicit about the link between the sentence before and this one.